nautobot/extras/models/customfields.py

```diff
@@ -124,6 +130,22 @@ def cf(self):
         """
         return self._custom_field_data
 
+    def get_basic_custom_fields(self):
+        """
+        Return a dictionary of custom fields for a single object in the form {<field>: value}
+        which have advanced_ui set to False
+        """
+        fields = CustomField.objects.get_for_model(self).filter(advanced_ui=False)
+        return OrderedDict([(field, self.cf.get(field.name)) for field in fields])
+
+    def get_advanced_custom_fields(self):
```

Comment:
It's unfortunate from a self-consistency perspective that we have:

```
get_computed_fields(advanced_ui=...)
get_relationships(advanced_ui=...)
```

but not:

```
get_custom_fields(advanced_ui=...)
```

Can we make this consistent with the other two, even if it means adjusting how the various content templates need to be implemented and/or adding a few more templatetags as translation layers?